.travis.yml

```diff
@@ -23,10 +23,8 @@ install:
 
 # Command to run tests, e.g. python setup.py test
 script:
-    - set -e # Exit on first failure (errexit)
```

Comment:
@mostafajahanifar Are you sure about this change?
I think @John-P fixed this by doing flake8 test first. Please can you check this or we should use the develop version of .travis.yml